tools/pwsh/powershell_helpers.ps1

```diff
@@ -52,10 +52,14 @@ function Invoke-In-Dir {
         C:\Users\Public\Documents
         C:\Users\Public\
 #>
-  param([string]$directory, [string]$command)
+  param([string]$directory, [string]$command, [switch]$noPrint)
```

Comment:
I'm not sure how much value the `noPrint` parameter adds - it adds complexity for marginal gains. 
Whether it's needed or not, it is missing from the function documentation.